src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -6,10 +6,23 @@
 */
 
 #include "rc_verbs.h"
+#include "rc_verbs_impl.h"
 
 #include <ucs/arch/bitops.h>
 #include <uct/ib/base/ib_log.h>
 
+void uct_rc_verbs_txcnt_init(uct_rc_verbs_txcnt_t *txcnt)
+{
+    txcnt->pi = txcnt->ci = 0;
+}
+
+void uct_rc_verbs_common_packet_dump(uct_base_iface_t *iface, uct_am_trace_type_t type,
```

Comment:
It's not a `common` now, need some other name.
Also you can remove the last parameter from `uct_rc_ep_packet_dump` and use it directly in accelerated verbs (would need to rename it), because no TM support is left in pure verbs transports.
Here you would need to use some other function which does not check `tmh_opcode` in RC header, because you do not need this field anymore.